### PR TITLE
Fix/ignore linting errors with newer pylint

### PR DESCRIPTION
**Pull request by felixfontein**

### CODE REVIEW

**Comment by felixfontein** [comment]
CI errors are unrelated.

File: src/antsibull_core/venv.py [hunk 1]
```diff
@@ -102,4 +102,5 @@ def install_package(package_name: str) -> sh.RunningCommand:
             directly to :command:`pip install`.
         :returns: An :sh:obj:`sh.RunningCommand` for the pip output.
         """
+        # pylint:disable-next=broad-exception-raised
         raise Exception('Not implemented')
```
Comment (gotmax23) [inline src/antsibull_core/venv.py:106]: This should raise a NotImplementedError

File: src/antsibull_core/vendored/collections.py [hunk 2]
```diff
@@ -103,6 +103,7 @@ def count(seq):
     deprecated and replaced when support for Python < 2.7 is dropped.
     """
     if not is_iterable(seq):
+        # pylint:disable-next=broad-exception-raised
         raise Exception('Argument provided  is not an iterable')
```
Comment (gotmax23) [inline src/antsibull_core/vendored/collections.py:107]: This should raise a TypeError, but I guess we don't want to modify vendored code

File: src/antsibull_core/venv.py (hunk 1 above)
Comment (felixfontein) [reply src/antsibull_core/venv.py:106]: That's not a change that should happen in this PR.

**Pull request merged**